_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/templates/node/node--connectors--list-view.html.twig

```diff
@@ -0,0 +1,21 @@
+{% set classes = [
+  'pf-l-grid',
+  'pf-m-gutter',
+  'product-version'
+]%}
+<div{{ attributes.addClass(classes) }}{{ audience_selection }}>
+  <div class="pf-l-grid__item product-version__file-label-wrapper">
+    <p class="product-version__file-label">{{ label }}</p>
```

Comment:
The original ticket states: "...displaying the value in the Title field as a hyperlink to the URL in the Connector Details URL field..."
It seems like this should link out.

Reply:
Ah, you are right. Thank you!

Reply:
This looks good.